### ISSUE TITLE
Use more precise language

### ISSUE THREAD

**Issue by jodischneider** [zuphilip/zotero-open-citations#1]
I'm not sure what "Add some functionalities for open citations" means. Does this mean that you are finding citations to articles from OCC based on the DOI? More clear language would help potential users.

**Comment by zuphilip** [comment]
Yeah, I agree, that seems not very understandable. Let me try again:

"This Zotero plugin adds information about open citations data for one or several items in Zotero. Currently, a DOI lookup in OpenCitations is implemented and the result is added as a note:"

Better? Okay?

**Comment by jodischneider** [comment]
"This Zotero plugin looks up one or several items in OpenCitations by DOI. For each selected item, it adds a note indicating the number of citations found in OpenCitations."

**Comment by zuphilip** [comment]
Thank you @jodischneider, but the pull request was merged two years ago. Have a look at the new version of the text in the current [README](https://github.com/zuphilip/zotero-open-citations/blob/master/README.md) and let me know if there is any further clarification you would like to change there.
